docs/examples/game-of-chance.md

```diff
@@ -1,271 +1,150 @@
 ---
-sidebar_label: Game of chance
 sidebar_position: 9
 ---
 
 # Game of chance
 
 ## Introduction
 
-Anyone can easily create their own game application and run it on the Gear Network. To do this, Gear created an example of the Game-of-chance smart contract, which is available on [GitHub](https://github.com/gear-dapps/lottery). 
+:::note
 
-This article explains the programming interface, data structure, basic functions their purpose. It can be used as is or modified to suit your own scenarios.
+This article explains at a superficial level the purpose and logic of this smart contract. For a more detailed technical description, see its [documentation on the dApps documentation portal](https://dapps.gear.rs/game_of_chance) and [the source code section](#source-code).
```

Comment:
the link does not work
https://dapps.gear.rs/game_of_chance

Reply:
This link will work after the docs have been fixed, will do it today.

Reply:
Done.